src/System.Globalization/tests/CompareInfo/CompareInfoTests.IndexOf.cs

```diff
@@ -61,30 +59,15 @@ public static IEnumerable<object[]> IndexOf_TestData()
             yield return new object[] { s_invariantCompare, "More Test's", "Tests", 0, 11, CompareOptions.IgnoreSymbols, 5 };
             yield return new object[] { s_invariantCompare, "More Test's", "Tests", 0, 11, CompareOptions.None, -1 };
             yield return new object[] { s_invariantCompare, "cbabababdbaba", "ab", 0, 13, CompareOptions.None, 2 };
-        }
 
-        public static IEnumerable<object[]> IndexOf_Random_TestData()
-        {
             string[] interestingStrings = new string[] { "", "a", "1", "-", "A", "!", "abc", "aBc", "a\u0400Bc", "I", "i", "\u0130", "\u0131", "A", "\uFF21", "\uFE57" };
             foreach (string string1 in interestingStrings)
             {
                 foreach (string string2 in interestingStrings)
                 {
-                    yield return new object[] { s_currentCompare, string1, string2, 0, string1.Length, CompareOptions.Ordinal, PredictIndexOfOrdinalResult(string1, string2) };
+                    yield return new object[] { s_currentCompare, string1, string2, 0, string1.Length, CompareOptions.Ordinal, string1.IndexOf(string2, StringComparison.Ordinal) };
```

Comment:
I realize this is what the original test was doing, but this seems busted to me.  We're testing indexOf by telling it the expected position is the result of IndexOf.

What I'd intended in my comments was to just add some entries manually based on these inputs, like the rest of the test data.